Sources/Adapters/Tensorflow/TFConverter+Mappers.swift

```diff
@@ -101,8 +101,14 @@ public extension TFConverter {
             kernelHeight: weightData.weightShape.kernelHeight,
             strideX: Int(strides.x),
             strideY: Int(strides.y))
+
+        var weights = weightData.weights
+        if #available(iOS 11.3.1, *) {
```

Comment:
What about adding a param to `ConvTranspose` to tell to use MPS' transpose conv, so the condition to use it (`#available(iOS 11.3.1, *)`) is located only in one place?

Reply:
The #available needs to be there for the code to compile

Reply:
Yeah, but what about checking it only in one place and saving it, like `useMPSConv = #available(iOS 11.3.1, *)`?

Reply:
Oh, good to know!